common.yml

```diff
@@ -767,6 +767,8 @@ pre:
           export AWS_SECRET_ACCESS_KEY='${release_aws_secret}'
           export NOTARY_KEY_NAME='server-4.4'
           export NOTARY_TOKEN='${signing_auth_token_server_4_4}'
```

Comment:
since these are now set on a per-curator-call basis in the go code, it seems like we _shouldn't_ have these variables set by default in the evg yml
```suggestion
```